edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -33,56 +32,54 @@ public class EdgeAgentConnection : IEdgeAgentConnection
         readonly Task initTask;
         readonly RetryStrategy retryStrategy;
         readonly PeriodicTask refreshTwinTask;
-        readonly IRequestManager requestManager;
-        readonly IStreamRequestListener streamRequestListener;
+        readonly IModuleConnection moduleConnection;
+        readonly bool pullOnReconnect;
```

Comment:
Is this part of the bandwidth reductions?

Reply:
Yes, that is correct. In fact that is why this PR was initially created (because we have to close the ModuleClient after creating it to avoid bytes on the wire when idle)